presto-druid/src/main/java/com/facebook/presto/druid/DruidBrokerPageSource.java

```diff
@@ -79,6 +78,8 @@ public DruidBrokerPageSource(
                 .map(DruidColumnHandle::getColumnType)
                 .collect(toImmutableList());
         this.pageBuilder = new PageBuilder(this.columnTypes);
+        this.columnHandlesHasErrorMessageField = columnHandles.stream().anyMatch(
```

Comment:
do we need to keep columnHandlesHasErrorMessageField? how about we inline it in getNextPage()